### ISSUE TITLE
WalletConnect images are not loading

### ISSUE THREAD

**Issue by ignaciosantise** [CypherD-IO/mobile-app#295]
### Summary
* Wallet logos are not loading in the app

### Steps to reproduce:
1. Open Cypher Wallet
2. Press "Connect A Wallet" button
3. See that wallet logos are not loading

Tested on:
* iOS 17 - Cypher Wallet v2.55
* Android 14 - Cypher Wallet v4.06

#### Android:
<img src="https://github.com/CypherD-IO/mobile-app/assets/25931366/4a911850-96a3-43d6-9b12-1254be3a6040" height="700" >

#### iOS:
<img src="https://github.com/CypherD-IO/mobile-app/assets/25931366/98d51955-9eb8-4dea-98d4-ecef013e1637" height="700" >

Probably you can solve the issue by upgrading `@web3modal/coinbase-react-native`  `@web3modal/wagmi-react-native` to the latest version.

Happy to help if the issue still happens after the update 🫡

**Comment by ignaciosantise** [comment]
@shan-c

**Comment by shan-c** [comment]
HI @ignaciosantise. Thanks for the suggestion. We are using @web3modal/wagmi-react-native 2.0.0 alpha. Do you recommend using 1.4.1

**Comment by ignaciosantise** [comment]
@shan-c you're welcome 🤝 

the new alpha version adds support for wagmi 2, so i recommend it if you currently use wagmi v2.

Also, if interested, we recently added email login -> https://docs.walletconnect.com/appkit/react-native/onboarding/email

**Comment by shan-c** [comment]
Yes, We've recently migrated to wagmi v2. We will also explore email login.

**Comment by ignaciosantise** [comment]
@shan-c the correct version of the packages is:

```
    "@walletconnect/core": "2.15.1",
    "@walletconnect/react-native-compat": "2.15.1",
    "@walletconnect/utils": "2.15.1",
    "@walletconnect/web3wallet": "1.14.1",
    "@web3modal/coinbase-wagmi-react-native": "2.0.2",
    "@web3modal/wagmi-react-native": "2.0.2",
```

Also, you're missing the final steps to enable Coinbase support -> check the docs: https://docs.walletconnect.com/appkit/react-native/core/installation#enable-coinbase-wallet
